### ISSUE TITLE
Create application form for instructor training

### ISSUE THREAD

**Issue by gvwilson** [swcarpentry/amy#784]
Using the following as a starting point, create a form people can use to apply for instructor training.

* Relates to #599: form for asking for instructor training event.
* Closes #419: import old data (we'll ask people to re-apply).

Note: we'll want to match applicants to existing Person records, but *cannot* show them possible matches to agree to while they're applying (leaks data, violates privacy).

~~~
Thanks to generous sponsorship from the R Consortium, Software
Carpentry is running a two-day R instructor training class in
Cambridge, UK, on September 19-20, 2016. If you are active in the R
and/or Software and Data Carpentry communities, and wish to apply to
attend this event, please fill in the form below. We will select
applicants, and notify everyone who applied, by June 30, 2016. If you
have any questions, please mail r-training-class@software-carpentry.org.

Please note that as a condition of taking this training:

1. You are required to abide by our code of conduct, which can be
found at http://software-carpentry.org/conduct/.

2. You must complete three short tasks after the course in order to
complete certification. The tasks are described at
http://swcarpentry.github.io/instructor-training/checkout/, and take a
total of approximately 2 hours.

3. You are expected to teach at a Software Carpentry or Data Carpentry
workshop within 12 months of the course.

Name
as you are normally addressed, without title

Email

Institutional Affiliation
current company, university, research lab, etc.

Location
please give city, province or state, and country

Domain/Specialization
* Statistics/Mathematics
* Computer Science
* Physical Science (astronomy, physics, chemistry...)
* Biological Science (zoology, physiology, genetics...)
* Social Science (economics, anthropology...)
* Humanities (history, architecture...)
* Library Science
* Other :

Career Stage
* Undergraduate
* Graduate/Post-doctoral
* Faculty
* Industry
* Other :

Gender
This information will be used solely to help Software Carpentry and Data Carpentry foster diversity within their communities.
* Female
* Male
* Other
* Prefer not to say

Previous Involvement with Software/Data Carpentry
* Learner
* Helper
* Instructor
* Workshop host/organizer
* Contributed to lesson materials
* Other:

Previous Training in Teaching
* None
* A few hours
* A few days
* A full degree
* Other :

Previous Experience in Teaching
* None
* Have taught for a few hours
* Have taught entire courses
* Other :

How Frequently Do You Use R?
* Every waking moment
* Hourly
* Daily
* Weekly
* Not much

Why Do You Want to Attend This Training Course?

How Often Would You Expect to Teach Classes on R and/or Software or Data Carpentry Workshops After This Training?
* Not at all
* Once a year
* Several times a year
* Primary occupation
* Other :

How Frequently Would You Be Able to Travel to Teach Such Classes?
* Not at all
* Once a year
* Several times a year
* Other :

Do You Have Any Additional Relevant Skills or Interests That We Should Know About?
For example, do you have experience creating video courses or teaching people with disabilities, or are you a member of an under-represented group or have experience working with under-represented groups?

I Agree to Abide by Software Carpentry's Code of Conduct
The Code of Conduct can be found at http://software-carpentry.org/conduct/
* I agree

I Agree to Complete This Training Within Three Months of the Training Course
The completion steps are described at http://swcarpentry.github.io/instructor-training/checkout/ and take a total of approximately two hours.
* I agree

I Agree to Help Teach a Software Carpentry or Data Carpentry Workshop Within 12 Months of This Training Course
* I agree
~~~

**Comment by chrismedrela** [comment]
Please review: https://github.com/swcarpentry/amy/pull/809

**Comment by pbanaszkiewicz** [comment]
Closing since it was merged in #809.
